### DISCUSSION TITLE
Link on one of my pages not working?

### DISCUSSION THREAD

**Discussion by ivanuricardo** [quarto-dev/quarto-cli#11546] [Q&A] [answered]
### Description

Hello, 

I currently have a [website](https://ivanuricardo.github.io/) which all works, except on the [software page](https://ivanuricardo.github.io/software.html), I have a link to one of my projects which does not work. If i try clicking it, simply nothing happens. What is more confusing is if I go to my [publications](https://ivanuricardo.github.io/publications.html), the link to my arxiv paper and github pages works. Moreover, it is not like the link is broken or leads to a different page, but rather, I simply can not click on the link.

For some more information, I am developing on Linux EndeavourOS (arch 6.12.1) and the quarto version I have is 1.5.57. You can find the source code for my website at [this link](https://github.com/ivanuricardo/ivanuricardo.github.io). Thank you for any suggestions!

**Comment by cderv** [comment]
This is because somehow the header part is taking to much height, and it gets over your content. 
![image](https://github.com/user-attachments/assets/21cb9bd8-20fc-48d2-bbc3-904b0869aa1a)

The layering means that when you hover the link with your mouse, it does not see the link. It considers the header. 

If I put something like 
```css
main {
  z-index: 100;
}
```

it works again. 

Did you tweak your website ? or could it be a problem with default Quarto website ? 

Thanks

**Reply by cderv** [reply, marked as answer]
I believe this is your custom styling having an impact here
https://github.com/ivanuricardo/ivanuricardo.github.io/blob/c4a06e7371797639778b1b5c423da56faf0f8a10/software.qmd#L7-L23

If you remove it, the link is accessible.

**Reply by ivanuricardo** [reply]
Oh perfect, thank you! I will then use this to fix it.
